deployment/ccip/changeset/state.go

```diff
@@ -249,6 +249,16 @@ func (c CCIPChainState) GenerateView() (view.ChainView, error) {
 		}
 		chainView.TokenAdminRegistry[c.TokenAdminRegistry.Address().Hex()] = taView
 	}
+	for version, tokenPool := range c.USDCTokenPools {
```

Comment:
let's skip this I have a Pr for this for all pool formatting - https://github.com/smartcontractkit/chainlink/pull/16572